### ISSUE TITLE
HTML Import polyfill bug with absolute paths on Safari 11, but not firefox

### ISSUE THREAD

**Issue by pdesjardins90** [webcomponents/webcomponentsjs#855]
When using absolute paths for HTML imports, like this:
```html
<!-- /index.html -->
<link rel="import"
      href="/src/first-component.html">
<link rel="import"
      href=/src/second-component.html">
```

If second-component.html imports first-component.html too, like this: 
```html
<!-- /src/second-component.html -->
<link rel="import"
      href="/src/first-component.html">
```

It will execute any script it includes a second time, instead of just recognizing it has already been loaded. 

If using relative paths, like this:
```html
<!-- /index.html -->
<link rel="import"
      href="src/first-component.html"
<link rel="import"
      href="src/second-component.html">
```

```html
<!-- /src/second-component.html -->
<link rel="import"
      href="first-component.html">
```

It will execute first-component's script only once, like it should. 

- This doesn't happen on chrome (native HTML import).
- This doesn't happen on firefox (who requires the polyfill).

**Comment by valdrinkoshi** [comment]
Looks like a duplicate of https://github.com/webcomponents/html-imports/issues/72, which was fixed in ``html-imports v1.0.3` & `webcomponentsjs v1.0.14` 👍
